src/cmd/run.js

```diff
@@ -312,14 +310,18 @@ export default async function run(
         log.info('Press R to reload (and Ctrl-C to quit)');
         if (process.stdin instanceof tty.ReadStream) {
           process.stdin.setRawMode(true);
+          process.stdin.on('keypress', (str, key) => {
+            if (key.ctrl && key.name === 'c') {
+              if (process.stdin instanceof tty.ReadStream) {
```

Comment:
@saintsebastian that's weird, but I think that there is a better workaround:
we can extract the `process.stdin` property into its own `const` first and then used it in the code that follows, that should convince flow that `stdin` is not going to change its type, something like:

```js
      const {stdin} = process;
      if (stdin.isTTY && stdin instanceof tty.ReadStream) {
        readline.emitKeypressEvents(process.stdin);
        log.info('Press R to reload (and Ctrl-C to quit)');
        stdin.setRawMode(true);
        stdin.on('keypress', (str, key) => {
          if (key.ctrl && key.name === 'c') {
            exitProgram({firefox: runningFirefox, stdin});
          } else if (key.name === 'r' && addonId) {
            addonReload({addonId, client});
          }
        });
      }
```

How that sounds to you?